RtspServer/MediaTypes/RFC2435Media.cs

```diff
@@ -2317,15 +2317,8 @@ public RFC2435Media(string name, string directory, bool watch, int width, int he
 
         void EnsureDimensions()
         {
-            int over;
-
-            Math.DivRem(Width, Common.Binary.BitsPerByte, out over);
-
-            /*if (over > 0) */Width += over;
-
-            Math.DivRem(Height, Common.Binary.BitsPerByte, out over);
-
-            /*if (over > 0) */Height += over;
+            Width += Width % Common.Binary.BitsPerByte;
```

Comment:
It used to be hard to get DivRem to be called but this still calls it per sharplab..

Thank you!

https://sharplab.io/#v2:C4LghgzgtgNAJiA1AHwAICYCMBYAUKgZgAIMiBhIgbzyNqIEsA7YIgdXrmAAsYiAJAKb0A5l2ABuGnUIkALEQCyACgCUVKXU3tOXIogC8bDtyIBSIgA5JuTbcEixew/dEtzVjUQC+nzzNTyAHKqntQ2trRMLAD2AG4CAE7WnhG0CmDcAHQAIvSxAEoCUEra3LwWvNEArjHxCSrJ4al0APQAVPQAZkRKcYlEAHxEAAxqbS2lugZEfUl4KanpWbkFRUouYuWVNTN1DfNNzUTtXT2zgyNjLRss07PWmj64XkA==